extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -132,6 +132,13 @@ describes.realWin('amp-ad-network-adsense-impl', {
       impl = new AmpAdNetworkAdsenseImpl(element);
       expect(impl.isValidElement()).to.be.true;
     });
+    it('should be NOT valid (wrong responsive ad height)', () => {
```

Comment:
Can you add a test that verifies that isValidElement will return true when data-auto-format=rsvp and height=320?